README.md

```diff
@@ -71,29 +49,25 @@ docker volume create ubuntu-tezos-volume
 ```
 
 This docker image has [`./scripts/docker.sh`](./scripts/docker.sh) as an entrypoint.
-This script basically wraps [`build-patched-binaries.sh`](./scripts/build-patched-binaries.sh)
+This script basically wraps [`fetch-binaries.sh`](./scripts/fetch-binaries.sh)
 and [`start-baker.sh`](./scripts/start-baker.sh) scripts providing required paths
 for tezos-binaries stored inside the docker volume.
 
-## Generating new genesis public key and building patched binaries
+## Generating new genesis public key and fetching binaries
 
 ### Without docker
 
 First step for running the private blockchain is generating a new genesis public key and
-building patched Tezos binaries. [`build-patched-binaries.sh`](./scripts/build-patched-binaries.sh)
-shell script will build these patched binaries.
-
-Note that you may have to adjust this script or [`patch_template.patch`](./patches/patch_template.patch)
-for your needs, e.g. genesis public keys can be moved from
-`src/proto_genesis_{babylonnet, carthagenet}/lib_protocol/data.ml` to another files.
+fetching Tezos binaries. [`fetch-binaries.sh`](./scripts/fetch-binaries.sh)
```

Comment:
Just changing the wording a bit:

fetching Tezos binaries. The [`fetch-binaries.sh`](./scripts/fetch-binaries.sh)
shell script will build these patched binaries.
This shell script will fetch statically built binaries and create a tezos-node config file
containing a custom genesis key.  This custom genesis key will be used by the script `start-baker.sh`.